Frameworks/include/CGColorSpaceInternal.h

```diff
@@ -18,14 +18,4 @@
 
 #import <objc/runtime.h>
 
-class __CGColorSpace : private objc_object {
```

Comment:
it is in CGColorSpace :)

---
In reply to: [106516141](https://github.com/Microsoft/WinObjC/pull/2253#discussion_r106516141) [](ancestors = 106516141)